ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -167,12 +172,24 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate) {
     // Strip metadata so that it's not subject to transformation.
     Object metadata = tableRow.remove(AddMetadata.METADATA);
 
+    String namespace = message.getAttribute(ParseUri.DOCUMENT_NAMESPACE);
+    String docType = message.getAttribute(ParseUri.DOCUMENT_TYPE);
+    final boolean strictSchema;
+    if (("telemetry".equals(namespace) && strictSchemaDocTypes.get().contains(docType))
```

Comment:
This special-casing for telemetry seems unnecessarily, we could remove it (which would simplify the documentation above and the implementation here).

Reply:
Perhaps in retrospect that would have been a good idea. We already support this format for other options that take in a comma separated list of docTypes, though, so I'm inclined to keep consistent.

Reply:
You're right. I've changed this.